src/log_rotate_manager.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Log rotate manager."""
+
+import logging
+import os
+import signal
+import subprocess
+import typing
+
+from ops.framework import Object
+from ops.model import ActiveStatus
+
+if typing.TYPE_CHECKING:
+    from charm import MySQLOperatorCharm
+
+logger = logging.getLogger(__name__)
+
+# File path where log rotate manager exceptions will be stored
+LOG_FILE_PATH = "/var/log/log_rotate_manager.log"
```

Comment:
Yes, we're just logging python exceptions here. If we need to rotate these logs as well, I'd rather do it in a separate PR (although since the code is tested, I dont expect a lot of logs to accumulate in these files). I'll ask @taurus-forever when he's back about whether it is required

Reply:
actually, it seems like these log files were created mainly for development - redirecting the output to devnull instead in 7c2a5c87cb1aefad61a67e333da1099948bf18b0